core/global_constants.cpp

```diff
@@ -607,8 +607,23 @@ void register_global_constants() {
 	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_COLOR_NO_ALPHA);
 	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_IMAGE_COMPRESS_LOSSY);
 	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_IMAGE_COMPRESS_LOSSLESS);
-
+	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_OBJECT_ID);
+	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_TYPE_STRING);
+
+	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_NODE_PATH_TO_EDITED_NODE);
+	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_METHOD_OF_VARIANT_TYPE);
+	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_METHOD_OF_BASE_TYPE);
+	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_METHOD_OF_INSTANCE);
+	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_METHOD_OF_SCRIPT);
+	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_PROPERTY_OF_VARIANT_TYPE);
+	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_PROPERTY_OF_BASE_TYPE);
+	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_PROPERTY_OF_INSTANCE);
+	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_PROPERTY_OF_SCRIPT);
+	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_OBJECT_TOO_BIG);
+	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_NODE_PATH_VALID_TYPES);
+	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_HINT_SAVE_FILE);
```

Comment:
Missing `PROPERTY_HINT_ENUM_SUGGESTION`.

Reply:
And the docs check will likely still fail as `PROPERTY_HINT_MAX` is bound but not documented.
The best is to run `--doctool` to generate the template from your compiled binary.

Reply:
Ah I see. I guess it was put there because `ENUM` and `ENUM_SUGGESTION` are linked, but were not made consecutive in the enum to avoid breaking compat. Can probably be kept as is.